src/app/components/phase-steps/group-feedback/group-feedback.component.html

```diff
@@ -4,11 +4,11 @@
     <div fxLayout>
       <div fxFlex="50%">
         <h2>Groups</h2>
-        <md-input-container>
-          <input mdInput placeholder="Enter a new group name" [(ngModel)]="newGroupName" name="newGroupNameInput" (keyup.enter)="createNewGroup(newGroupName)"/>
+        <md-input-container *ngIf="allowAdminFunctionsLocal === true">
```

Comment:
Shouldn't this just be *ngIf="allowAdminFunctions"? I don't see an allowAdminFunctionsLocal variable in the group-feedback component.